lib/ansible/modules/cloud/docker/docker_service.py

```diff
@@ -511,18 +511,17 @@ def make_redirection_tempfiles():
 
 
 def cleanup_redirection_tempfiles(out_name, err_name):
-    get_redirected_output(out_name)
-    get_redirected_output(err_name)
+    for i in [out_name, err_name]:
+        os.remove(i)
 
 
 def get_redirected_output(path_name):
     output = []
     with open(path_name, 'r') as fd:
         for line in fd:
             # strip terminal format/color chars
-            new_line = re.sub(r'\x1b\[.+m', '', line.encode('ascii'))
+            new_line = re.sub(r'\x1b\[.+m', '', to_text(line))
```

Comment:
`for line in fd` return line as `str` and `re.sub` take `str` as third argument, so there's no need to call `to_text` here.